client/cpu_linux.go

```diff
@@ -18,3 +18,7 @@ func osflags(fi os.FileInfo, mode p9.OpenFlags) int {
 	}
 	return flags
 }
+
+func osDefaultCpudCmd() string {
```

Comment:
fwiw: these could also be 
const osDefaultCpudCmd = "/proc/self/exe -remote"
for example, i.e. no need to be a function, wdyt?

Reply:
done, thanks